code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -5090,6 +5090,71 @@
 	if(!H.isDead())
 		updateconsumingalt(equipped_slot == slot_wear_mask)
 
+/obj/item/weapon/reagent_containers/food/snacks/lollicheap
```

Comment:
make a subtype instead of pasting all this!

Reply:
I'll probably have to refactor it for the normal medipops too, honestly. Which wouldn't be a terrible idea, in retrospect.